src/ListInline.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { mapToCssModules, tagPropType } from './utils';
+
+const propTypes = {
+    tag: tagPropType,
+    className: PropTypes.string,
+    cssModule: PropTypes.object
+};
+
+const defaultProps ={
+    tag: 'ul'
+};
+
+const ListInline = (props) => {
```

Comment:
Let's make this component a little bit more generic so we can cover `.list-unstyled` here too, for example:
`<List type="unstyled">` and `<List type="inline">`

Also use a `forwardRef` here